stile/hsc/base_tasks.py

```diff
@@ -1310,6 +1323,12 @@ class MultiTractSingleEpochStileTask(TractSingleEpochStileTask):
     _DefaultName = "MultiTractSingleEpochStile"
     ConfigClass = TractSingleEpochStileConfig
     multi_item_type='tract'
+    item_type = 'patch'
```

Comment:
Can I ask why these 5 lines have been duplicated?  Since `MultiTractSingleEpochStileTask` inherits from `TractSingleEpochStileTask`, they should already be defined.  I'd rather not duplicate this code (more places to look for bugs/updates) without a reason.